presto-accumulo/src/test/java/com/facebook/presto/accumulo/TestAccumuloDistributedQueries.java

```
@@ -0,0 +1,503 @@
+//                "test_union_all",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders WHERE orderkey % 2 = 0 UNION ALL " +
+//                        "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders WHERE orderkey % 2 = 1",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders",
+//                "SELECT count(*) FROM orders");
+
+//        Function "UUID" not found
+//        this.assertCreateTableAsSelect(
+//                getSession().withSystemProperty("redistribute_writes", "true"),
+//                "test_union_all",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT count(*) + 1 FROM orders");
+
+//        Function "UUID" not found
+//        this.assertCreateTableAsSelect(
+//                getSession().withSystemProperty("redistribute_writes", "false"),
+//                "test_union_all",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT count(*) + 1 FROM orders");
+    }
+
+    @Override
+    public void testDelete()
+            throws Exception
+    {
+        try {
+            // TODO Deletes are not supported by the connector
+            super.testDelete();
+        }
+        catch (Exception e) {
+            assertUpdate("DROP TABLE test_delete");
```

Maybe put it in `finally` block, similarly to how it's done in `AbstractTestHiveClient`?
Make sure to not swallow exception revealing potential test failure, so please long accordingly in `finally` block.